Sources/BraveWallet/Panels/Sign Transaction/SignTransactionView.swift

```diff
@@ -0,0 +1,52 @@
+// Copyright 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import BraveCore
+import DesignSystem
+import SwiftUI
+
+struct SignTransactionView: View {
+  enum Request {
+    case signTransaction(BraveWallet.SignTransactionRequest)
+    case signAllTransactions(BraveWallet.SignAllTransactionsRequest)
+  }
+  
+  var request: Request
+  var cryptoStore: CryptoStore
+  var onDismiss: () -> Void
+  
+  var navigationTitle: String {
+    switch request {
+    case .signTransaction:
+      return "Sign Transaction"
+    case .signAllTransactions:
+      return "Sign All Transactions"
```

Comment:
Do we want these translated?

Reply:
Yeah I can localize these here, but otherwise would be handled in #6005

Reply:
https://github.com/brave/brave-ios/pull/6050/commits/87993ce8e4d3855c9e0ac2f522d60c999d6d337f